components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.ts

```diff
@@ -151,6 +153,8 @@ export class DashboardComponent implements OnInit {
       this.store.select(unknownDesktopDurationCounts).pipe(
       map(counts => counts.updated)
     );
+
+    setTimeout(() => this.layoutFacade.hideSidebar());
```

Comment:
Used the same `setTimeout` workaround from https://github.com/chef/automate/pull/3082/commits/9eb7496997d9e77ba642b65be9db6f5f9f4568c4 to avoid:

<img width="980" alt="Screen Shot 2020-04-20 at 12 30 24 PM" src="https://user-images.githubusercontent.com/479121/79787027-91a6f500-8303-11ea-8c9c-5bd7e930f760.png">

Need to investigate the implementation and usage of `layoutFacade` to find a way a better way around this particular issue.